provisioning/local/azure.tf

```diff
@@ -0,0 +1,61 @@
+locals {
+  azure_location = "West Europe"
+}
+
+variable "az_tenant_id" {
+  type    = string
+  default = "9b85ee6f-4fb0-4a46-8cb7-4dcc6b262a89" # Keboola
+}
+
+provider "azurerm" {
+  features {}
+  tenant_id       = var.az_tenant_id
+  subscription_id = "c5182964-8dca-42c8-a77a-fa2a3c6946ea" # Keboola DEV Platform Services Team
```

Comment:
asi bych dal do locals, jako máme jinde

Reply:
s tim jsme myslim v locals zacali, ale pak prisla potreba mit referencovany dve subscription a zacal v tom byt nejak bordel.. jelikoz to neni nikde reusnuty (to subscription ID je v celym provisioningu 1x), nechal bych to klidne primo v tom provideru jako hodnotu